purchases/src/main/java/com/revenuecat/purchases/interfaces/LogInListener.java

```diff
@@ -0,0 +1,31 @@
+//  Purchases
+//
+//  Copyright © 2019 RevenueCat, Inc. All rights reserved.
+//
+
+package com.revenuecat.purchases.interfaces;
+
+
+import androidx.annotation.NonNull;
+
+import com.revenuecat.purchases.PurchaserInfo;
+import com.revenuecat.purchases.PurchasesError;
+
+/**
+ * Interface to be implemented when calling logIn
+ */
+public interface LogInListener {
```

Comment:
I am all for calling this `LogInCallback` instead. Also if you make it extend `PurchaseErrorListener`, it will have the     `void onError(@NonNull PurchasesError error);`. I would also do it in Kotlin too. I don't remember the reason why I originally did all the other interfaces in Java, I think it was something related to Java compatibility, but I just tried this case and a kotlin and a java interface look the same when using our SDK in a java app

Reply:
love moving it to kotlin. Turns out the IDE has an auto-convert option 🤯 
I'm not sure about extending `PurchaseErrorListener`, though: 
an error logging in isn't technically a purchase error, and it would also add an extra param (userCancelled) that won't make sense: 
```
    void onError(@NonNull PurchasesError error, boolean userCancelled);
```

Reply:
Yes, the `PurchaseErrorListener` part didn't make sense. My bad. I forgot it was for callbacks from purchasing